Source/Characteristic.swift

```diff
@@ -40,7 +40,7 @@ public class Characteristic {
     /// Unique identifier of an object. Should be removed in 4.0
     @available(*, deprecated)
     public var objectId: UInt {
-        return characteristic.objectId
+        return UInt(bitPattern: ObjectIdentifier(characteristic))
```

Comment:
Maybe we should allow to to get CBCharacteristic from Characteristic? If we will be sticking to objectId API, maybe it's better to just return ObjectIdentifier(...) as it is Equatable and Hashable.